src/pages/public_registration/LogPanel.tsx

```diff
@@ -59,21 +103,29 @@ export const LogPanel = ({ tickets, registration }: LogPanelProps) => {
           <Typography>{new Date(registration.createdDate).toLocaleDateString()}</Typography>
         </StyledStatusMessageBox>
       )}
-      {registration.publishedDate && (
-        <StyledStatusMessageBox sx={{ bgcolor: 'publishingRequest.main' }}>
-          <Typography>{t('registration.status.PUBLISHED_METADATA')}</Typography>
-          <Typography>{new Date(registration.publishedDate).toLocaleDateString()}</Typography>
-        </StyledStatusMessageBox>
-      )}
-      {tickets
-        .filter((ticket) => ticketStatusesToShow.includes(ticket.status) && ticketTypesToShow.includes(ticket.type))
-        .sort((a, b) => +a.modifiedDate - +b.modifiedDate)
-        .map((ticket) => {
-          if (ticket.type === 'PublishingRequest') {
-            return <CompletedPublishingRequestStatusBox key={ticket.id} ticket={ticket as PublishingTicket} />;
+      {logs
+        .filter(
+          (logEntry) =>
+            isNonTicketLogEntry(logEntry) ||
+            (isTicket(logEntry) &&
+              ticketStatusesToShow.includes(logEntry.status) &&
+              ticketTypesToShow.includes(logEntry.type))
+        )
+        .sort((a, b) => new Date(a.modifiedDate).getTime() - new Date(b.modifiedDate).getTime())
+        .map((logEntry, index) => {
+          if (isTicket(logEntry) && logEntry.type === 'PublishingRequest') {
+            return <CompletedPublishingRequestStatusBox key={logEntry.id} ticket={logEntry as PublishingTicket} />;
+          }
+          if (isTicket(logEntry) && logEntry.type === 'DoiRequest') {
+            return <DoiRequestMessagesColumn key={logEntry.id} ticket={logEntry} />;
           }
```

Comment:
Vurderer litt å droppe gjenbruk av disse komponentene, og heller behandle alle entries i logs-listen. Legge på et felt som sier hvilken type/bakgrunsfarge innslaget skal ha. Er jo ikke en komplisert komponent som gjenbrukes. Men litt dumt å ikke gjenbruke allikevel... 
Har du noen tanker? Synes det er litt rotete med disse `if`-sjekkene i `map`-funksjonen.

Reply:
Sant det, egentlig er det bare `<StyledStatusMessageBox>` som vi ønsker å gjenbruke? Prøv det som gir mest mening for deg, så tipper jeg det blir veldig bra jeg!

Reply:
Ble litt kode og kompleksitet der også ser jeg, men syns det ble bedre ja! 🙌